hbase-client/src/main/java/org/apache/hadoop/hbase/client/AsyncConnectionImpl.java

```diff
@@ -203,26 +204,36 @@ public boolean isClosed() {
     return closed.get();
   }
 
+  private void closeQuietly(Closeable c) {
```

Comment:
The old implementation just suppress the exception without logging. For guava Closeables, it will never throw IOException if threw is false.

So the suggestion is to upgrade the commons-io and uses the new methods?

Can revert here as it is not a big deal for the PR here.